inc/saiswitch.h

```diff
@@ -3197,6 +3197,18 @@ typedef enum _sai_switch_attr_t
      */
     SAI_SWITCH_ATTR_SHARED_BUFFER_CELL_SIZE,
 
+    /**
+     * @brief Global PTP mode configuration
+     *
+     * Global PTP mode configuration for the switch.
+     * Applies to all ports unless overridden by port-specific settings.
+     *
+     * @type sai_port_ptp_mode_t
+     * @flags CREATE_AND_SET
+     * @default SAI_PORT_PTP_MODE_NONE
+     */
+    SAI_SWITCH_ATTR_PTP_MODE,
```

Comment:
The sai meta checkers would expect this attribute to be named as SAI_SWITCH_ATTR_PORT_PTP_MODE or the enum to be named as sai_ptp_mode_t.